### ISSUE TITLE
Injecting in only one html file

### ISSUE THREAD

**Issue by bisk8s** [icelam/html-inline-script-webpack-plugin#335]
Hi!
In html-webpack-plugin is possible to generate 2 files by using 2 plugins

```
plugins: [
      new HtmlWebpackPlugin({
        filename: 'index.webos.html',
        template: 'static/index.webos.html'
      }), // webos index - I need this one to be inline
      new HtmlWebpackPlugin(), // normal index -  would be better if this one was left alone
      new HtmlInlineScriptPlugin(),
    ]
```
there is a way to set your plugin only in the first index?

**Comment by icelam** [comment]
Hi @bisk8s, this plugin currently doesn't have logic that allow user to configure which HTML file they want the plugin to handle. However, the scenario you provided is a valid use case, which would be nice if this plugin can have some sort of ways to allow user to configure a include / exclude file list. 

I think I could be able to work on the enhancement within the next few days (or perhaps weeks) and release a new version. Meanwhile, one possible workaround might be [exporting multiple webpack configurations as documented on webpack's documentation](https://webpack.js.org/configuration/configuration-types/#exporting-multiple-configurations).

**Comment by icelam** [comment]
Hi @bisk8s, I have released a new version of this plugin (3.0.0) which includes this new feature. Let me know if this doesn't solve your issue.
